pkg/operator/collection.go

```diff
@@ -462,6 +469,41 @@ func (r *collectionReconciler) makeCollectorConfig(ctx context.Context, spec *mo
 		}
 	}
 
+	if err := r.client.List(ctx, &NodeMons); err != nil {
+		return nil, fmt.Errorf("failed to list NodeMonitorings: %w", err)
+	}
+	// The following job names are reserved by GMP for NodeMonitoring. They will not be generated if kubeletScraping is enabled.
```

Comment:
Hm, can we add a safeguard around namespaces? One could be surprised that certain names are blocked, although rare

Reply:
thanks! good idea!